pkgs/applications/networking/cluster/nova/default.nix

```
@@ -0,0 +1,38 @@
+{ lib, stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "nova";
+  version = "3.6.2";
+  arch = if stdenv.isx86_64
+    then "amd64"
+    else if stdenv.isAarch64 then "arm64"
+    else (assert false "Invalid architecture"; "");
+
+  sha256sum = if arch == "amd64"
+    then "31e6672dc94ca438fa2cb22c8f24173c9a91d0b443b4a9da2e64c6bffbdb7a81"
+    else if arch == "arm64"
+    then "4d993e3d975adeeb16c9b8b07558fd46b78f39e2900252243ebc4c845bdc26e9"
+    else (assert false "Invalid architecture"; "");
+
+  src = fetchurl {
+      url = "https://github.com/FairwindsOps/nova/releases/download/${version}/nova_${version}_linux_${arch}.tar.gz";
+      sha256 = sha256sum;
+    };
+
+  setSourceRoot = "sourceRoot=`pwd`";
+
+  installPhase = ''
+    tar -xvf $src
```

The unpacking is normally done in `unpackPhase` automatically.

```suggestion
  sourceRoot = ".";

  installPhase = ''
```